example/ios/Podfile.lock

```diff
@@ -10,37 +10,37 @@ PODS:
     - DatadogInternal (= 2.18.0)
   - DatadogRUM (2.18.0):
     - DatadogInternal (= 2.18.0)
-  - DatadogSDKReactNative (2.4.4):
+  - DatadogSDKReactNative (2.5.0):
     - DatadogCore (~> 2.18.0)
     - DatadogCrashReporting (~> 2.18.0)
     - DatadogLogs (~> 2.18.0)
     - DatadogRUM (~> 2.18.0)
     - DatadogTrace (~> 2.18.0)
     - DatadogWebViewTracking (~> 2.18.0)
     - React-Core
-  - DatadogSDKReactNative/Tests (2.4.4):
+  - DatadogSDKReactNative/Tests (2.5.0):
     - DatadogCore (~> 2.18.0)
     - DatadogCrashReporting (~> 2.18.0)
     - DatadogLogs (~> 2.18.0)
     - DatadogRUM (~> 2.18.0)
     - DatadogTrace (~> 2.18.0)
     - DatadogWebViewTracking (~> 2.18.0)
     - React-Core
-  - DatadogSDKReactNativeSessionReplay (2.4.4-alpha.0):
+  - DatadogSDKReactNativeSessionReplay (2.5.0):
     - DatadogSDKReactNative
     - DatadogSessionReplay (~> 2.18.0)
     - React-Core
-  - DatadogSDKReactNativeSessionReplay/Tests (2.4.4-alpha.0):
+  - DatadogSDKReactNativeSessionReplay/Tests (2.5.0):
```

Comment:
is it expected change? both are coming out of alpha?

Reply:
Not expected, it was not aligned with the latest change to revert the version back to alpha. We have just changed it.